lib/Perl/LanguageServer.pm

```diff
@@ -508,7 +524,7 @@ sub _run_tcp_server
                 if (!$guard && ($@ =~ /Address already in use/))
                     {
                     # stop other server
-                    tcp_connect '127.0.0.1', $listen_port, sub
```

Comment:
This should stay 127.0.0.1 because it tries to stop an already running server on the same host (which is actualy not always working)